### PR TITLE
add items from ENM pulling the strings

### PR DESCRIPTION
**Pull request by KnowOne134**

### CODE REVIEW

File: scripts/globals/items/assassins_ring.lua [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+-- ID: 14678
+-- Item: Assassin's Ring
+-- Item Effect: Ranged Accuracy 20
+-- Duration 3 Minutes
+-----------------------------------------
+require("scripts/globals/status")
+
+function onItemCheck(target)
+    local effect = target:getStatusEffect(dsp.effect.ENCHANTMENT)
+    if (effect ~= nil) then
```
[the remark this answers is not in the document]
Comment (KnowOne134) [reply scripts/globals/items/assassins_ring.lua]: i got no console errors

File: scripts/globals/items/vial_of_refresh_musk.lua [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
 
 function onItemUse(target)
-    if (not target:hasStatusEffect(dsp.effect.REFRESH)) then
-        target:addStatusEffect(dsp.effect.REFRESH,1,3,60)
+    if target:hasStatusEffect(dsp.effect.ENCHANTMENT) == true then
+        target:delStatusEffect(dsp.effect.ENCHANTMENT)
+        target:addStatusEffect(dsp.effect.ENCHANTMENT,0,0,30,18241)
     else
-        target:messageBasic(dsp.msg.basic.NO_EFFECT)
+        target:addStatusEffect(dsp.effect.ENCHANTMENT,0,0,30,18241)
```
Comment (TeoTwawki) [inline scripts/globals/items/vial_of_refresh_musk.lua]: erm, both condition branches add the status effect? just verifying this is what you intend here, and what retail does.

Comment (KnowOne134) [reply scripts/globals/items/vial_of_refresh_musk.lua]: Just based on this comment i found on ffxiah.com. its a very rare item as the rest of them. The only one i have personal experience is with blm item. 
https://www.ffxiah.com/item/18241/refresh-musk

File: scripts/globals/items/astral_pot.lua [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    if not pet then
+        return dsp.msg.basic.REQUIRES_A_PET,0
+    elseif effect ~= nil and effect:getSubType() == 18243 then
+        target:delStatusEffect(dsp.effect.ENCHANTMENT)
+    end
+    return 0
+end
+
+function onItemUse(target)
+    if target:hasStatusEffect(dsp.effect.ENCHANTMENT) == true then
```
Comment (TeoTwawki) [inline scripts/globals/items/astral_pot.lua]: missed this before, don't need the `== true` on it

**Comment by KnowOne134** [comment]
Made a big mistake in my repo by cross merging and instead of fixing i closed the repo and made a new fork not knowing i wouldnt be able to make changes to this pull. Can you guys merge this ill pull changes to my new repo and make the suggested changes? Atm none of the content is implemented so bad code here for a day wont matter.

**Comment by TeoTwawki** [comment]
I can rescue it to a new branch for you

**Comment by KnowOne134** [comment]
Ok and i can then pull that branch. Perfect

**Comment by TeoTwawki** [comment]
grab from rescue-a-pr branch, make the requested changes (styling and addEffectEx to fix the icon) and we'll worry about the fact enchantments need to be stackable at a later date.

**Pull request merged**